ConsistencyManager/ConsistencyManager.swift

```diff
@@ -95,6 +95,12 @@ open class ConsistencyManager {
     var listeners = [String: WeakListenerArray]()
 
     /**
+     This is an array of all the model update listeners.
+     These will get notified whenever anything changes in the Consistency Manager.
+     */
```

Comment:
Yeah, makes sense but we want to make it obvious that this is the solution. Both here as code doc but also probably in the doc site.